extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -126,13 +126,24 @@ private static function addLiftIgniterMetadata( OutputPage $outputPage ) {
 
 		$context = RequestContext::getMain();
 		$title = $context->getTitle();
+		$articleId = $title->getArticleID();
 		$metaData = [];
+		$metaDataService = new LiftigniterMetadataService();
+		$metaDataFromService = $metaDataService->getLiMetadataForArticle($wgCityId, $articleId);
 
 		if ( $title->isMainPage() ) {
 			$siteAttributeData = self::getLiftIgniterMetadataFromSiteAttributeService();
```

Comment:
I wonder what's stored there and maybe we should move these data to our service in future?